package.json

```diff
@@ -95,8 +95,8 @@
     "ember-cli-babel": "^5.1.7",
     "ember-cli-htmlbars": "^1.1.1",
     "ember-cli-sass": "5.6.0",
-    "ember-frost-core": "1.23.13",
-    "ember-frost-list": "5.8.2"
+    "ember-frost-core": "1.23.10",
+    "ember-frost-list": "6.1.2"
```

Comment:
Is this version bump necessary to implement this functionality?

Reply:
No, it's not necessary. Actually, I'm not sure why we need it at all as a dependency. Moving this to dev-dependency.